pkg/contract_watcher/parser/parser_test.go

```diff
@@ -117,109 +109,4 @@ var _ = Describe("Parser", func() {
 			Expect(ok).To(Equal(false))
 		})
 	})
```

Comment:
You don't have a test where you pass anything other than "" for the abiKey. Not sure if it matters.

Reply:
Yeah not really sure what a good alternative would be - I think adding that test would require that we have an actual Etherscan API key passed to run the tests, and I'm more tempted to go in the other direction (e.g. don't even require ipc path for unit tests). That said maybe it's worth creating a story for adding an integration tests that uses an API key, and that adding an encrypted var for that to travis?